Sysvale/component-library
# feature/creates-badge-component

#### What is the purpose of this pull request?
To create the Badge component as the first component in the library

#### How should this be manually tested?
After cloning the repository and installing the dependencies, run `npm run storybook` and see if the storybook pages opens and the component is redered properly and you can interact with it using the addons in the right panel.

#### Screenshots or example usage
![image](https://user-images.githubusercontent.com/20057968/69854819-cccb5d80-1268-11ea-810d-1a61c43bfaf1.png)

#### Types of changes
